Replaces goto with go-to for consistency

bruno-oliveira opened the pull request:
Unifies commits

bruno-oliveira commented:
@sbauer322 Is this what you intended? I hope I did it right!

sbauer322 commented:
Yes, @bruno-oliveira, that is about right - thank you! All the changes appear in this pull request, which is desired and much easier to review.

It looks like some extra files were added, namely the ones ending with a `~`. Could you remove those as they are unnecessary?

sbauer322 commented on src/lt/objs/cli.cljs~:
```diff
@@ -0,0 +1,75 @@
+(ns lt.objs.cli
```
This file can be removed.

sbauer322 commented on src/lt/objs/find.cljs~:
```diff
@@ -0,0 +1,229 @@
+(ns lt.objs.find
```
This file can be removed.

sbauer322 commented on src/lt/objs/search.cljs~:
```diff
@@ -0,0 +1,277 @@
+(ns lt.objs.search
```
This file can be removed.

sbauer322 commented on src/lt/objs/find.cljs:
```diff
@@ -215,8 +215,8 @@
-(cmd/command {:command :goto-line
-              :desc "Editor: Goto line"
+(cmd/command {:command :go-to-line
+              :desc "Editor: go to line"
```
This line should have a capital 'G' for consistency... so `:desc "Editor: Go to line"`

bruno-oliveira commented:
Hello, 
I have cleaned up the repository and also changed the "go to line" to "Go to line" for consistency, as requested.

I hope everything is well now, and if this is merged, I'd love to start helping with other issues suitable for beginners!

sbauer322 commented:
Tested it out and there is one more necessary minor change.

As the command `:goto-line` was renamed, it will be necessary to update the corresponding default keymap entry to use `:go-to-line` instead. This is located at `deploy/settings/default/default.keymap` on line 26.

Otherwise, everything seems in order.

bruno-oliveira commented:
Hello,
I'll update the keymap as soon as possible! Definitely today still!

Em 31/12/2016 03:18, "Scott Bauer" <notifications@github.com> escreveu:

> Tested it out and there is one more necessary minor change.
>
> As the command :goto-line was renamed, it will be necessary to update the
> corresponding default keymap entry to use :go-to-line instead. This is
> located at deploy/settings/default/default.keymap on line 26.
>
> Otherwise, everything seems in order.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/LightTable/LightTable/pull/2306#issuecomment-269846309>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEgO7I2BelZGn4gYbs1LniD4qOSp_jA2ks5rNclpgaJpZM4LX6-Y>
> .
>

bruno-oliveira commented:
Hello @sbauer322 ,
I have just updated the keymap as requested and also deleted an additional file that was generated ending in ~. I hope everything is finally in order! I'll be happy to fix whatever needs fixing still!

bruno-oliveira commented:
Any good news? :)

sbauer322 commented:
This looks good to me - `Ctrl-L` is working again! Thank you for your efforts!

The final thing you will need to do is squash your commits into a single commit.

I am going to leave this open until January 4th in order to give other @LightTable/maintainers a chance to look it over, should they decide to.

bruno-oliveira commented:
Hello,
I think I have squashed all previous commits into one, as requested... Is it all correct?

sbauer322 commented:
Yes, that seems correct. I hate to keep picking, but could you update the commit message to some more meaningful? When it gets merged to master, the current commit message won't be very helpful down the line.

sbauer322 commented:
Looks like it is in great shape!

bruno-oliveira commented:
That's good to know! :D Thanks for all the feedback and help during the process

bruno-oliveira commented:
Any updates on the state of merging? :)
I hope all is good, but, if I still need to do something, I will gladly do it!

sbauer322 commented:
Simply was giving anyone an opportunity to provide feedback... it is never a bad idea to have additional eyes on code!

Since no one has provided any additional feedback I will be merging this tomorrow, January 5th.
